test/system/init-contracts.js

```diff
@@ -0,0 +1,158 @@
+const {
+  keepTokenAddress,
+  nuCypherTokenAddress,
+  keepTokenStakingAddress,
+  nuCypherStakingEscrowAddress,
+  keepRegistryAddress,
+} = require("./constants.js")
+const {
+  to1e18,
+  impersonateAccount,
+} = require("../helpers/contract-test-helpers")
+
+async function initContracts() {
+  const deployer = await ethers.getSigner(0)
+
+  const keepToken = await resolveKeepToken()
+  const nuCypherToken = await resolveNuCypherToken()
+  const keepTokenStaking = await resolveKeepTokenStaking()
+  const nuCypherStakingEscrow = await resolveNuCypherStakingEscrow()
+  const keepRegistry = await resolveKeepRegistry()
+
+  const tToken = await deployTToken(to1e18(1000000))
```

Comment:
Should be `10 000 000 000`. Non blocking but we should update it at some point to use real-world value.